pcsx2/Counters.cpp

```diff
@@ -202,6 +202,10 @@ static void vSyncInfoCalc(vSyncTimingInfo* info, Fixed100 framesPerSecond, u32 s
 
 	u64 Blank = HalfFrame / 2; // PAL VBlank Period is off for roughly 22 HSyncs
```

Comment:
Would be nice if you edited this please :)

Comment out the whole line please, and replace the original comment with "broken code, used before 01.02.2019" (or similar).
Then just use the correct line below for u64 Blank = ..